ui/dashboards/src/context/DashboardProvider/DashboardProvider.tsx

```diff
@@ -64,7 +66,8 @@ export function DashboardProvider(props: DashboardProviderProps) {
           ...createPanelGroupSlice(...args),
           ...createLayoutSlice(layouts)(...args),
           ...createPanelEditorSlice(panels)(...args),
-          defaultTimeRange: dashboardSpec.duration,
+          selectedTimeRange: selectedTimeRange ?? { pastDuration: dashboardSpec.duration },
```

Comment:
Do we need `selectedTimeRange` and `setSelectedTimeRange` in the store, or can we have these states live in the `TimeRangeProvider`?

Reply:
It's just that now the TimeRangeProvider depends on DashboardProvider so wondering if this is necessary

Reply:
Yeah, +1 to this question.

Reply:
Yeah I went back and forth on this, I think I got thrown off when I saw useDefaultTimeRange in dashboard-provider-api, but good point about not wanting to add a dependency to TimeRangeProvider. I've removed selectedTimeRange 
 and setSelectedTimeRange now and updated this PR with hopefully a much better approach!